apps/ewallet_db/lib/ewallet_db/wallet.ex

```diff
@@ -111,8 +111,12 @@ defmodule EWalletDB.Wallet do
     |> put_change(:encryption_version, Cloak.version())
   end
 
-  def all_for(%Account{} = account), do: from(t in Wallet, where: t.account_uuid == ^account.uuid)
-  def all_for(%User{} = user), do: from(t in Wallet, where: t.user_uuid == ^user.uuid)
+  def all_for(%Account{} = account),
```

Comment:
Sirn despises `..., do:` format that extends multiple line. Can we change to the usual multiliner?

Reply:
It's the `mix format`, urgh...